src/ReactLeafletMap/layers/TiledVectorLayer/Tile.js

```diff
@@ -89,19 +91,31 @@ class Tile extends React.PureComponent {
 	}
 
 	render() {
-		const {tileKey, featureKeysGroupedByTileKey, ...props} = this.props;
+		const {tileKey, featureKeysGroupedByTileKey, component, ...props} = this.props;
```

Comment:
Add component to propTypes as required?

Reply:
e0a7d3e2f03c00f2dfc62368f290ebebdc622270